.github/workflows/unit_tests.yml

```diff
@@ -31,4 +31,4 @@ jobs:
         run: pip install -e ".[dev]"
         if: steps.python-cache.outputs.cache-hit != 'true'
       - name: Run unit tests
-        run: pytest
+        run: tox
```

Comment:
@ColeDCrawford I don't know how to reconcile tox with the python version test matrix, do you ?

Reply:
Looks like `tox-gh-actions` worked!

Reply:
Agreed, it would be nice to pull the build matrix from `tox.ini tox.envlist` or `setup.cfg`, but not sure how to set that up.